testing/trino-testing/src/main/java/io/trino/testing/AbstractTestFailureRecovery.java

```diff
@@ -71,6 +71,8 @@
     protected abstract QueryRunner createQueryRunner(List<TpchTable<?>> requiredTpchTables, Map<String, String> configProperties, Map<String, String> coordinatorProperties)
             throws Exception;
 
+    protected abstract boolean areWriteRetriesSupported();
```

Comment:
Technically we do not. I should probably remove this whole block of tests and readd it later. But I do not believe it would be cleaner that way.  I would live it.